### ISSUE TITLE
Downloads very large, not stripped

### ISSUE THREAD

**Issue by probonopd** [AppImage/AppImageUpdate#69]
Please `strip` everything. Example:

```
squashfs-root/usr/bin/AppImageUpdate-Qt: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), 
dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.24, 
BuildID[sha1]=3f3deedac2cec5d4741965a605a7cd9db929300d, not stripped
```

**Comment by TheAssassin** [comment]
The point not to strip the binaries is it makes it way easier to debug our (still beta) software. Users will see stacktraces, and can provide them when asked to do so.

I am not sure where this overhead comes from, though. It's most likely not that the binaries aren't stripped (strip isn't perfect btw, e.g., static archives embedded aren't perfectly optimized, it'd be better to do that before linking the final executable).

I'll look into reducing the overhead. AppImageUpdate's binary used to be maybe around 5-6 MiB.

**Comment by TheAssassin** [comment]
The reason the main AppImage (FLTK UI) is so large is that all the Qt libraries have been bundled in it, too, unnecessarily. I know why that happens, and I will push a fix.

**Comment by TheAssassin** [comment]
WIP in #80.

**Comment by TheAssassin** [comment]
Closed via #80.
